geopandas/geodataframe.py

```diff
@@ -401,6 +401,10 @@ def to_file(self, filename, driver="ESRI Shapefile", schema=None,
         schema : dict, default: None
             If specified, the schema dictionary is passed to Fiona to
             better control how the file is written.
+        encoding : string, default: None
+            If specified, the encoding is passed to Fiona open method.
+            Otherwise, the drivers used by Fiona will try to detect the
+            encoding of data files.
```

Comment:
Just wondering, since this is in our `to_file` writing method (while `fiona.open` is both for reading and writing), does the  "detect the encoding of the data files" make sense for writing?